README.md

```diff
@@ -170,6 +170,8 @@ In general, we follow the "fork-and-pull" Git workflow.
 2. Clone the project to your own machine
 3. Work on your fork
     1. Make your changes and additions
+        - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`. 
+        - Files such as `validator.js`, `validator.nin.js` and files in `lib/` folder are autogenrated when running tests and need not to be changed `manually`. 
```

Comment:
Fix the typo, `validator.min.js`. Then you don't have to back-tick the word `manually`, perhaps you could _emp/italize_.

Reply:
I have made updates @profnandaa.